core/src/test/java/org/dcsa/conformance/core/check/JsonAttributeTest.java

```diff
@@ -0,0 +1,732 @@
+                JsonAttribute.isNotNull(JsonPointer.compile("/test3")),
+                JsonAttribute.mustBePresent(JsonPointer.compile("/test2")))
+            .validate(objectNode, "")
+            .isEmpty());
+  }
+
+  static Stream<Arguments> testIsNotNullArgs() {
+    return Stream.of(
+        Arguments.of("test isNotNull", JsonNodeFactory.instance.objectNode(), "/test", false),
+        Arguments.of(
+            "test isNotNull with existing value",
+            JsonNodeFactory.instance.objectNode().put("test", "test"),
+            "/test",
+            true));
+  }
+
+  @ParameterizedTest(name = "[{index}] {0}")
+  @MethodSource("testIsNotNullArgs")
+  void testIsNotNull(String testName, ObjectNode input, String path, boolean expected) {
+    assertEquals(expected, JsonAttribute.isNotNull(JsonPointer.compile(path)).test(input));
+  }
+
+  @Test
+  void testMustBeNotNull() {
+    ObjectNode objectNode = JsonNodeFactory.instance.objectNode();
+    objectNode.put("test", "test");
+    String reason = "test reason";
+
+    assertTrue(
+        JsonAttribute.mustBeNotNull(JsonPointer.compile("/test"), reason)
+            .validate(objectNode)
+            .isEmpty());
+    assertFalse(
+        JsonAttribute.mustBeNotNull(JsonPointer.compile("/test1"), reason)
+            .validate(objectNode)
+            .isEmpty());
+  }
+
+  @Test
+  void testMustEqual() {
+    ObjectNode objectNode = JsonNodeFactory.instance.objectNode();
+    objectNode.put("test", "test");
+
+    assertTrue(
+        JsonAttribute.mustEqual(JsonPointer.compile("/test"), "test")
+            .validate(objectNode)
+            .isEmpty());
+    assertFalse(
+        JsonAttribute.mustEqual(JsonPointer.compile("/test"), "test1")
+            .validate(objectNode)
+            .isEmpty());
+    Supplier<String> supplier = () -> "test";
+
+    assertTrue(
+        JsonAttribute.mustEqual(JsonPointer.compile("/test"), supplier)
+            .validate(objectNode)
+            .isEmpty());
+    supplier = () -> "test1";
+    assertFalse(
+        JsonAttribute.mustEqual(JsonPointer.compile("/test"), supplier)
+            .validate(objectNode)
+            .isEmpty());
+  }
+
+  @Test
+  void testPath() {
+    ObjectNode objectNode = JsonNodeFactory.instance.objectNode();
+    objectNode.put("test", "test");
+
+    assertTrue(
+        JsonAttribute.path("test", JsonAttribute.matchedMustBePresent())
+            .validate(objectNode, "")
+            .isEmpty());
+    assertFalse(
+        JsonAttribute.path("testFalse", JsonAttribute.matchedMustBePresent())
+            .validate(objectNode, "")
+            .isEmpty());
+  }
+
+  @Test
+  void testLostAttributeCheck() {
+    ObjectNode objectNode = JsonNodeFactory.instance.objectNode();
+    objectNode.put("test", "test");
+    Supplier<JsonNode> supplier = () -> objectNode;
+
+    assertDoesNotThrow(
+        () -> org.dcsa.conformance.core.check.JsonAttribute.lostAttributeCheck("test", supplier));
```

Comment:
Is the full package reference needed? Same for line 240.
```suggestion
        () -> JsonAttribute.lostAttributeCheck("test", supplier));
```